packages/dev/test/provider/RelayClient.test.ts

```diff
@@ -156,6 +157,7 @@ contract('RelayClient', function (accounts) {
     const loggerConfiguration: LoggerConfiguration = { logLevel: 'debug' }
     gsnConfig = {
       loggerConfiguration,
+      performDryRunViewRelayCall: false,
```

Comment:
so you run all tests with dryRun = false, and only a single test with dryRun=true ..
Since we want dry-run to be the default mode, it seems that it need to be reversed.